tests/unit_tests/forward_model_runner/test_jobmanager.py

```diff
@@ -122,6 +122,31 @@ def test_run_multiple_ok():
         assert os.path.getsize(f"mkdir_err.{dir_number}") == 0
 
 
+@pytest.mark.usefixtures("use_tmpdir")
```

Comment:
There should be 2-3 more tests:

* If the forward model steps do not produce a file specified in the manifest then the user should get a failed realization with a nice error message.
* If a file with the expected checksum is present for the forward model runner but not for scheduler a warning should be displayed followed by an error from the ok callback when there is no file and no error when there is a file ok_callback can load.